pages/index.vue

```diff
@@ -117,7 +117,7 @@
                   <h3 class="headline">Mit Kreditkarte bezahlen</h3>
                   <p>
                     Sie brauchen keine Rechnung abzuwarten, um mit dem Signieren
-                    loszulegen.
+                    los zulegen.
```

Comment:
```suggestion
                    loszulegen.
```
Duden says Firefox spellchecker needs to go back to school :)